salt/utils/__init__.py

```diff
@@ -1920,3 +1921,45 @@ def output_profile(pr, stats_path='/tmp/stats', stop=False, id_=None):
         stacklevel=3
     )
     return salt.utils.profile.output_profile(pr, stats_path, stop, id_)
+
```

Comment:
As noted in the header of this file, we are not adding new functions here. I don't see a natural home in the existing modules under `salt/utils/`, so this may need to go into its own file. `salt/utils/environment.py` perhaps?

Reply:
@terminalmage who reads the headers... my bad. But yeah, I thought about exactly this but hesitated. OK, `salt/utils/environment.py` it is.